src/integration/pages/mediaAssetPage/arabicTC2/amp.test.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * @service arabic
+ * @pathname /arabic/multimedia/2014/05/140515_kenia_children_sex_trade
```

Comment:
Can we have a different example?

Reply:
Good spot, thank you. Will get it updated asap,